build.mill

```diff
@@ -823,26 +848,6 @@ object dist extends MillPublishJavaModule {
     PathRef(T.dest / raw.last)
   }
 
-  def prependShellScript = T {
```

Comment:
Why is this moved to a local position? If you don't want the caching, you could make it a anonymous task instead.

Reply:
Dealt with in https://github.com/com-lihaoyi/mill/pull/3608